src/playdate/build/pdxinfo.nim

```diff
@@ -0,0 +1,55 @@
+import std/[os, parsecfg, streams, strutils, strformat, times, osproc], nimbledump
+
+type PdxInfo* = object
+    ## Details used to populate the pdxinfo file
+    name*, author*, description*, bundleId*, imagePath*, version*, buildNumber*: string
+
+proc `$`*(pdx: PdxInfo): string =
+    for key, value in pdx.fieldPairs:
+        if value != "":
+            result &= key & "=" & value & "\n"
+
+proc write*(pdx: PdxInfo) =
+    ## Writes the pdxinfo file
+    createDir("source")
+    writeFile("source" / "pdxinfo", $pdx)
+
+proc join*(a, b: PdxInfo): PdxInfo =
+    ## Combins two PdxInfo instances
+    result = a
+    for current, override in fields(result, b):
+        if override != "":
+            current = override
+
+proc parsePdx*(data: Stream, filename: string): PdxInfo =
```

Comment:
Is this reachable from game code? Might be interesting to make it so. To print the version number in the corner of the game's start screen.

... I'm guessing not since Stream is probably not available on device

Reply:
Totally to do that in a later PR or not all tbh.

Reply:
Would be trivial to pass it in as a define when calling nimble during the build

Reply:
(I can follow up with another PR to handle that)